portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -13,6 +13,6 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
The PR message says this is for `/home`, but the web sevlet path is still `/hello`, should it be changed to `/home`?

Reply:
Sorry, it should be for /hello